### ISSUE TITLE
URL signature expired

### ISSUE THREAD

**Issue by rzuf1k** [scottsweb/wp-instagram-widget#100]
Hello again! I am receiving this message when trying to show 3 out of 6 images. This is visible on: http://www.lionsestate.pl/en/ . Can You please check it?

**Comment by scottsweb** [comment]
Images seem to be OK for me.

**Comment by rzuf1k** [comment]
Hello, any update on this?

**Comment by scottsweb** [comment]
As I said before the images are loading for me:

![screenshot-20180731144055-587x633](https://user-images.githubusercontent.com/411945/43463122-c9dcf86c-94cf-11e8-9d2c-2f1d84d312cf.png)

Perhaps test in other browsers and with a different theme. Try to isolate some JavaScript on your site that me causing you a problem.

**Comment by rzuf1k** [comment]
White images are css background-image placeholders. If image is loaded correctly then img src overlaps background-image. If image is not loaded correctly then white img is displayed.

TL;DR
No, not all images are loaded correctly.

**Comment by rzuf1k** [comment]
Other computer, other network, other browser: 
![image](https://user-images.githubusercontent.com/5220468/43463798-c4bf7594-94d9-11e8-9f39-cae48cd88f0d.png)

**Comment by scottsweb** [comment]
How about trying a different theme? you can test one in the customiser without activating it.

**Comment by rzuf1k** [comment]
![image](https://user-images.githubusercontent.com/5220468/43474841-4325d814-94f4-11e8-905b-87fb6529e6f7.png)
Different theme. Thing is: some photos are visible when I am switching size of images. When I am selecting "original" size - 4 are visible. On thumbnail - only 2

**Comment by rzuf1k** [comment]
it is exact the same issue You resolved here: https://github.com/scottsweb/wp-instagram-widget/issues/81

**Comment by feastdesignco** [comment]
Using inspect element, we can see that it's 403'd.

Copying the image link directly into browser returns the error "URL signature expired".

It looks like lionsestate fixed this by returning a default image on failure - could be a good fall-back option.

Also, server-side caching could help avoid this: https://github.com/scottsweb/wp-instagram-widget/issues/90
